### PR TITLE
[stable/redis] support for setting up ClusterIP

### PR DESCRIPTION
**Pull request by cronolio**
hello. i like to specify ClusterIP for pods.
I beginning to be familiar with helm , please review my PR

### CODE REVIEW

File: stable/redis/templates/metrics-svc.yaml [hunk 1]
```diff
@@ -14,6 +14,10 @@ metadata:
 {{- end }}
 spec:
   type: {{ .Values.metrics.service.type }}
+  {{ if eq .Values.metrics.service.type "ClusterIP" -}} {{ if .Values.metrics.service.ClusterIP -}}
```
Comment (tompizmor) [inline stable/redis/templates/metrics-svc.yaml]: can't you use `and` and use a single `if` statement?

```
{{- if and (eq .Values.metrics.service.type "ClusterIP") (.Values.metrics.service.ClusterIP) }}
    ClusterIP: {{ .Values.metrics.service.ClusterIP }}
{{- end }}
```

File: stable/redis/values.yaml [hunk 2]
```diff
@@ -105,6 +105,7 @@ sentinel:
   service:
     ##  Redis Sentinel Service type
     type: ClusterIP
+    ClusterIP:
```
Comment (tompizmor) [inline stable/redis/values.yaml:108]: can you also add this same changes from the values.yaml to the values-production.yaml?

**Review by tompizmor — CHANGES_REQUESTED** [review]

File: stable/redis/Chart.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 8.1.1
+version: 8.1.2
```
Comment (tompizmor) [inline stable/redis/Chart.yaml]: Following semver versioning, you should update to 8.2.0 as you are introducing a new feature in the chart.

**Comment by tompizmor** [comment]
Please document also the new option in the README file. You should add it to the table.

**Comment by cronolio** [comment]
looks like all is done, pls review

File: stable/redis/templates/metrics-svc.yaml (hunk 1 above)
Comment (cronolio) [reply stable/redis/templates/metrics-svc.yaml]: i have done it for loadbalanser too

**Comment by tompizmor** [comment]
Hi @cronolio,

It seems you still need to do two things before this PR can be merged:

1. You need to fix the conflicts in the `stable/redis/Chart.yaml` file
2. You need to sign your commits. Check [this link](https://github.com/helm/charts/pull/15470/checks?check_run_id=167787690) to know what you have to do

**Comment by cronolio** [comment]
>     2. You need to sign your commits. Check [this link](https://github.com/helm/charts/pull/15470/checks?check_run_id=167787690) to know what you have to do

Hello, how can i do this via github website interface?

**Pull request closed without merging**